workshops/static/img/q2workshops.svg

```diff
@@ -1,3 +1,10 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2021, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
```

Comment:
I don't think this file is actually checked by q2lint, so this can be removed. Also, its an SVG file, so I think the right way to do comments are using the `<!-- -->` syntax.